docs/src/man/methods.md

```diff
@@ -19,16 +19,16 @@ In order to avoid having to specify this everywhere in the document, it is usefu
 With these notations we can write:
 
 ```math
-\begin{eqnarray}
-    S &=& \kappa^{-1}X_c^T X_c \label{simple-covariance}\\
-    D_{ij} &=& S_{ij}\mathbf 1_{[i=j]}\\
-    X_s &=& X_c D^{-1/2}
-\end{eqnarray}
+\begin{aligned}
+    S &= \kappa^{-1}X_c^T X_c\\
+    D_{ij} &= S_{ij}\mathbf 1_{[i=j]}\\
+    X_s &= X_c D^{-1/2}
+\end{aligned}
```

Comment:
nit: while this difference makes sense in LaTeX document where `aligned` is strongly recommended, it doesn't online where the reasons why align is better than eqnarray aren't relevant, and in fact online `eqnarray` is preferred because the spacing is more sensible.

Reply:
ignore this, I see that it was not working in the docs, thanks for picking this up and apologies for the comment!

Reply:
Yep - similarly the label didn't seem to be supported (hence the tweak below)!